### PR TITLE
Install instructions

### PR DESCRIPTION
**Pull request by YaronMiro**
@amitaibu
I think `elm` node package is missing from the instructions.
Only after installing this package I was able to execute `elm-package install -y` command.

### CODE REVIEW

**Comment by amitaibu** [comment]
Thanks. Actually It would be better to use the native installers - so the link should be to http://elm-lang.org/install

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "gulp-plumber": "1.0.1",
     "gulp-rev-all": "^0.7.5",
     "gulp-rev-replace": "^0.3.1",
-    "gulp-sass": "^1.0.0",
+    "gulp-sass": "^2.0.4",
```
Comment (amitaibu) [inline package.json]: can this be in a another PR please

**Comment by YaronMiro** [comment]
@amitaibu  Fixed ^^

**Comment by amitaibu** [comment]
Merged, thanks.
